src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -768,6 +768,8 @@ public override void Init(ICellEditorData owningEditor, bool fresh)
             behaviourEditor.Visible = false;
         }
 
+        endosymbiosisPopup.Lawk = Editor.CurrentGame.GameWorld.WorldSettings.LAWK;
```

Comment:
I think it would be better to put this configuration just before the endosymbiosis popup is opened, that way the value is always up to date when the popup is opened.